docker-compose.yml

```diff
@@ -100,37 +84,48 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
-  ferretdb-sqlite-replset-secured:
-    image: ${FERRETDB_IMAGE:-ferretdb-local}
+  postgres:
+    image: ${POSTGRES_IMAGE:-postgres:16.4}
+    restart: on-failure
+    environment:
+      # UTC−03:30/−02:30. Set to catch timezone problems.
+      - TZ=America/St_Johns
+      - POSTGRES_USER=postgres_user
+      - POSTGRES_PASSWORD=postgres_password
+      - POSTGRES_DB=dance
+
+  ferretdb2:
+    build:
+      context: ./build/deps
+      dockerfile: ferretdb.Dockerfile
     restart: on-failure
+    depends_on: ["postgres-documentdb"]
     ports:
-      - 27004:27004
+      - 27003:27003
     environment:
       # Always UTC+05:45. Set to catch timezone problems.
       - TZ=Asia/Kathmandu
       - FERRETDB_LOG_LEVEL=error
-      - FERRETDB_LISTEN_ADDR=:27004
-      - FERRETDB_HANDLER=sqlite
-      - FERRETDB_SQLITE_URL=file:/state/?_pragma=busy_timeout(20000)
-      - FERRETDB_TEST_ENABLE_NEW_AUTH=true
-      - FERRETDB_SETUP_USERNAME=username
-      - FERRETDB_SETUP_PASSWORD=password
-      - FERRETDB_SETUP_DATABASE=admin
-      - FERRETDB_REPL_SET_NAME=rs0
+      - FERRETDB_LISTEN_ADDR=:27003
+      - FERRETDB_HANDLER=postgresql
+      - FERRETDB_POSTGRESQL_URL=postgres://username:password@postgres-documentdb:5432/postgres?sslmode=disable
+      - FERRETDB_AUTH=false
+      # - FERRETDB_REPL_SET_NAME=rs0
     extra_hosts:
       - "host.docker.internal:host-gateway"
 
-  postgres:
-    image: ${POSTGRES_IMAGE:-postgres}
+  postgres-documentdb:
+    build:
+      context: ./build/deps
+      dockerfile: postgres.Dockerfile
+    platform: linux/amd64
     restart: on-failure
```

Comment:
We could use the same PostgreSQL container and image for both v1 and v2

Reply:
Don't we want to compare the performance of ferretdb v1 on postgresql 16?

Reply:
I don't think we want to compare PostgreSQL versions, at least not yet

Reply:
Cool, let's use one image of postgresql :+1: